dev-scripts/quick

```diff
@@ -11,6 +11,7 @@ TAG="${TAG:-$(grep -m1 ' TAG:' .github/workflows/pull-request.yml | sed -e 's/^[
 OS="${OS:-linux}"
 ARCH="${ARCH:-amd64}"
 REPO="${REPO:-rancher}"
+DEBUG=${DEBUG:-false}
```

Comment:
nit: `DEBUG` seems rather generic.. I'm thinking of moving helm chart build also inside the dockerfile, so in that case `DEBUG` could mean `--debug` in the helm command. Or maybe `DEBUG` could mean adding more verbosity to `docker build` command, etc.

Can you think of another name that better implies "I want to compile the binary with debugging flags"?

Reply:
6d6e20624